crates/solidity/testing/snapshots/bindings_output/expressions/outputs/input.sol

```diff
@@ -0,0 +1,24 @@
+contract Test {
```

Comment:
question: what is the "expression output" being tested here?

Reply:
This is testing that function call expressions "return" a type so that you can chain a member access. Similar for the `new` operator, which is then composed with a function call. Probably makes sense to split them in separate cases, and rename accordingly.